packages/jss-plugin-isolate/src/index.js

```diff
@@ -1,5 +1,13 @@
+// @flow
 import inheritedInitials from 'css-initials/inherited'
 import allInitials from 'css-initials/all'
+import type {Plugin} from 'jss'
+import type StyleRule from 'jss/src/rules/StyleRule'
```

Comment:
I think we should stop importing from non-public types, since we decided to export types. In this case we should either as well export all rule types or user the Rule type https://github.com/cssinjs/jss/blob/master/packages/jss/src/types/jss.js#L19

Reply:
Yeah, but I need to say that explicitly it's a StyleRule, so we need to export the rules as well

Reply:
https://github.com/cssinjs/jss/blob/master/packages/jss-plugin-syntax-rule-value-observable/src/index.js#L4 we have the same problem here

Reply:
Then we have to export them all I guess